allegro.go

```diff
@@ -69,7 +69,7 @@ func Index(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 func checkTimestamp(timestamp int64) error {
 	curTime := time.Now().UnixNano() / (int64(time.Millisecond) / int64(time.Nanosecond))
 	diff := curTime - timestamp
-	if diff < 0 || diff > (1000*60*60) {
+	if diff < -(1000*60*60) || diff > (1000*60*60) {
```

Comment:
Im fine with this. but we could take the absolute value as well. If nobody else cares neither do I.